benchpress/qiskit_gym/manipulate/test_manipulate.py

```diff
@@ -137,6 +137,30 @@ def result():
         assert result
 
 
+    def test_random_clifford_decompose(self, benchmark):
+        """Decompose a random clifford into
+        basis [rz, sx, x, cz]
+        """
+        translate = generate_preset_pass_manager(
+            1, basis_gates=["rz", "sx", "x", "cz"]
+        ).translation
+        cliff_circ = QuantumCircuit.from_qasm_file(
+            Configuration.get_qasm_dir("clifford") + "clifford_20_12345.qasm"
+        )
+        from qiskit.quantum_info import Clifford
+        cliff = Clifford(cliff_circ)
+        circ = cliff.to_circuit()
+
+        @benchmark
+        def result():
+            translate.property_set = PropertySet()
+            out = translate.run(circ)
+            return out
+
+        benchmark.extra_info["gate_count_2q"] = result.count_ops().get("cz", 0)
```

Comment:
Can you remove the hardcoding of the 2Q gate here as now done in the other tests.

Reply:
I see that this line also appears in other tests like in `test_QV100_basis_change`:
https://github.com/Qiskit/benchpress/blob/27b978f887d2187827c7792187df7f0d364b7114/benchpress/qiskit_gym/manipulate/test_manipulate.py#L136

Reply:
Yes, but again, will be removed when I get a chance.  Take a look at how it is done in the transpile tests

Reply:
I'm not sure how to do this. it's probably better to make a separate PR for all the `gate_count_2q` in this file.